README.md

```diff
@@ -49,6 +49,22 @@ tests are written in Kotlin and use the standard `kotlintest` testing framework.
 
     make integration
 
+## Run in an IDE                                                                                                                                                                              
+                                                                                                                                                                                              
+Both Kafka2S3 and the integration tests can be run in an IDE to facilitate                                                                                                                 
+quicker feedback then a containerized approach. This is useful during active development.                                                                                                     
+                                                                                                                                                                                              
+To do this first bring up the s3, kafka and zookeeper containers:                                                                                                                          
+                                                                                                                                                                                              
+    make services                                                                                                                                                                             
+                                                                                                                                                                                              
+Then insert into your local hosts file the names, IP addresses of the kafka and                                                                                                               
+hbase containers:                                                                                                                                                                             
+                                                                                                                                                                                              
+    ./hosts.sh                                     
```

Comment:
I don't see this file anywhere

Reply:
Looks like I forgot to stage that one. It's there now. Thanks for spotting that.

Reply:
maybe this should also be run through a `make` command, for consistency

Reply:
Done.